CharlieBackend.Business/Helpers/IJWTGenerator.cs

```diff
@@ -0,0 +1,9 @@
+﻿using CharlieBackend.Core.DTO.Account;
+
+namespace CharlieBackend.Business.Helpers
+{
+    public interface IJWTGenerator
```

Comment:
According to conventions, it should be `IJwtGenerator`. Also, don't forget to update implementation and fields.

Reply:
0214670